src/games/strategy/net/OpenFileUtility.java

```diff
@@ -0,0 +1,54 @@
+package games.strategy.net;
+
+import java.awt.Desktop;
+import java.io.File;
+import java.io.IOException;
+import java.net.URI;
+
+import javax.swing.JOptionPane;
+
+import games.strategy.debug.ClientLogger;
+
+public class OpenFileUtility {
```

Comment:
public classes should have a class comment so the intent of the class does not have to be reverse engineered from the code.  The previous one was overly worded and had unnecessary information, but a few words could be of some help to ensure there is understanding on why we would want to use this.